lib/services/cloud-codesign-service.ts

```diff
@@ -89,7 +89,12 @@ export class CloudCodesignService extends CloudService implements ICloudCodesign
 
 		if (!codesignResult.buildItems || !codesignResult.buildItems.length) {
 			// Something failed.
-			const err: any = new Error(`Codesign failed. Reason is: ${codesignResult.errors}.`);
+			let errText = codesignResult.errors;
+			if (codesignResult.errors.indexOf("403 Forbidden") > -1) {
+				errText = "The Code Signing Assistance service is temporary unavailable. Please try again later"
+			}
+
+			const err: any = new Error(`Codesign failed. Reason is: ${errText}.`);
```

Comment:
You can introduce a new type for this error and add the `stderr` in it. This way you'll not have to cast to any.